packages/style-engine/class-wp-style-engine.php

```diff
@@ -292,9 +282,6 @@ protected static function is_valid_style_value( $style_value ) {
 	 * Private constructor to prevent instantiation.
 	 */
 	private function __construct() {
-		foreach ( static::$stores as $store_key => $store_instance ) {
-			static::$stores[ $store_key ] = WP_Style_Engine_CSS_Rules_Store::get_store( $store_key );
-		}
```

Comment:
No longer needed since the `$stores` was removed. Instead of this, we'll be getting a store directly when needed.